.travis/install_z3_yices.sh

```diff
@@ -31,12 +31,16 @@ mkdir -p "$DOWNLOAD_DIR"
 cd "$DOWNLOAD_DIR"
 
 #download z3 and yices
+curl --retry 3 http://yices.csl.sri.com/releases/2.6.1/yices-2.6.1-x86_64-pc-linux-gnu-static-gmp.tar.gz --output yices.tar.gz
```

Comment:
Same here.  files should be in the s2n bucket to ensure they stay around.

Reply:
This link should now work: `https://s3-us-west-2.amazonaws.com/s2n-public-test-dependencies/yices-2.6.1-x86_64-pc-linux-gnu-static-gmp.tar.gz`